console/src/main/scripts/plugins/metrics/less/metrics.less

```diff
@@ -235,7 +251,7 @@ a:hover {
 
   .hk-graph-container  {
     margin: 0 auto;
-    //width: 750px;
+    width: 750px;
```

Comment:
Wouldn't this return us to fixed width graphs? And not the semi-responsive that we have now?